float64 probability
float64 distance
float64 x
float64 y
float64 width
float64 height
string name
